cdk/config.ts

```diff
@@ -16,6 +20,7 @@ export class Config {
       author: String(process.env.AUTHOR),
       gitCommit : String(process.env.COMMIT_SHA),
       gitRepository: String(process.env.GIT_REPOSITORY),
+      version: String(process.env.VERSION),
```

Comment:
let's remember to address how we version this deployment in another PR.

Reply:
I used this tag as a way to force cdk to redeploy - saw VEDA was doing something similar with their backend